integration/ide/gen-idea/resources/extended/idea/libraries/sbt_ junit_junit_3_6_4_13_2_jar.xml

```diff
@@ -1,5 +1,5 @@
 <component name="libraryTable">
-    <library name="SBT: junit:junit_3.6:4.13.2:jar">
+    <library name="sbt: junit:junit_3.6:4.13.2:jar">
```

Comment:
Don't know if those still have any special meaning, but I'd say let's keep the upper case versions in those technical use places.